octobot_trading/exchanges/abstract_websocket_exchange.py

```diff
@@ -195,10 +195,19 @@ def get_exchange_pair(self, pair):
     async def reset(self):
         raise NotImplementedError("reset is not implemented")
 
-    def add_pairs(self, pairs):
+    def add_pairs(self, pairs, watching=False):
```

Comment:
👍 
One quick idea: I think it would be better to name it `watching_only` or something with `only` considering that if this is true, it will not do as many things